.github/tests/dependencies/spire-root-server-values.yaml

```diff
@@ -6,7 +6,14 @@ global:
 
 spire-server:
   controllerManager:
-    enabled: false
+    identities:
+      namespaceSelector:
+        kubernetes.io/metadata.name: spire-server
```

Comment:
in the simple setup without namespace splitting we tend to use `spire-system`. What is the best default here?

Reply:
This is configuring the root spire server to provide an identity for the leaf spire-server-0 pod in the spire-server namespace.

Its a clean way to take the place of the hacky thing we had before:
https://github.com/spiffe/helm-charts-hardened/blob/main/examples/nested/run-tests.sh#L36-L38